### PR TITLE
Templates: Add new templates from batch 4

### LINKED ISSUE #7237: Templates Project: Batch 4 (31-40)
## Task Description

[**Figma link**](https://www.figma.com/file/OQIT5ISzdQVvRD7Aor8ajA/Web-Stories-50-Templates?node-id=1682%3A0)

Screenshot 2021-04-16 at 17 49 17

This batch includes 10 new templates:

1. Entertainment
2. Entertainment
3. Travel
4. Travel
5. Technology
6. Technology
7. Fashion & Beauty
8. Fashion & Beauty
8. Arts & Craft
9. Home & Garden

### PR DESCRIPTION
**Pull request by divyarajmasani**
## Summary

Adds 10 new story templates.

- New York Party Round-up
- A Day in the Life
- Elegant Travel Itinerary
- Modernist Travel Guide
- Simple Tech Tutorial 
- Magazine Article
- Fashion Inspiration
- Skin Care at Home
- Art Books Gift Guide
- Vintage Chair Buying Guide

Meta data for the templates were provided along with the designs in the Figma file itself, this includes sticker names, title, tags, description and colour palette.

## Relevant Technical Choices

- Adds stickers and story template JSON
- Uses title/alt texts for videos and images from the original source i.e. getty / unsplash / pexels.

## Testing Instructions

New templates should be available under 'Explore Templates' page.
`/wp-admin/edit.php?post_type=web-story&page=stories-dashboard#/templates-gallery`

Screenshot 2021-08-04 at 9 15 18 PM

## Checklist

- [x] This PR addresses an existing issue and I have linked this PR to it in ZenHub
- [x] I have tested this code to the best of my abilities
- [x] I have verified accessibility to the best of my abilities ([docs](https://github.com/google/web-stories-wp/blob/main/docs/accessibility-testing.md))
- [x] I have verified i18n and l10n (translation, right-to-left layout) to the best of my abilities
- [x] This PR contains automated tests (unit, integration, and/or e2e) to verify the code works as intended ([docs](https://github.com/google/web-stories-wp/tree/main/docs#testing))
- [x] I have added documentation where necessary
- [x] I have added a matching `Type: XYZ` label to the PR

---

Fixes #7237
Companion PR #8586

### CODE REVIEW

File: packages/stickers/src/a-day-in-the-life/dumbells.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
+/*
```
Comment (swissspidy) [inline packages/stickers/src/a-day-in-the-life/dumbells.js]: Isn't it `dumbbells` with two `b`?

File: packages/stickers/src/a-day-in-the-life/index.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * Internal dependencies
+ */
+import { default as dayInTheLifeBrwonRice } from './brownRice';
```
Comment (swissspidy) [inline packages/stickers/src/a-day-in-the-life/index.js]: `Brwon` -> `Brown`

File: packages/stickers/src/magazine-article/flightPath.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import { _x } from '@web-stories-wp/i18n';
+import PropTypes from 'prop-types';
+
+const title = _x('Flight path', 'sticker name', 'web-stories');
```
Comment (swissspidy) [inline packages/stickers/src/magazine-article/flightPath.js]:
```suggestion
const title = _x('Flight Path', 'sticker name', 'web-stories');
```

I think? Other sticker names are capitalized like this.

File: packages/templates/src/raw/a-day-in-the-life/index.js [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+    _x('Minimal', 'template keyword', 'web-stories'),
+    _x('Blue', 'template keyword', 'web-stories'),
+  ],
+  colors: [
+    { label: _x('Waterworld Blue', 'color', 'web-stories'), color: '#061b38' },
+    { label: _x('Arctic White', 'color', 'web-stories'), color: '#fcfcfc' },
+    { label: _x('Honey Pot Yellow', 'color', 'web-stories'), color: '#ffc864' },
+  ],
+  description: __(
+    'This simple and minimalist template will let you create long format video stories that are highly engaging. Create day-in-the-life videos, workout routines, make up tutorials and much more.',
```
Comment (swissspidy) [inline packages/templates/src/raw/a-day-in-the-life/index.js]: In some templates we write `makeup`, in others we write `make up`.

AFAIK `makeup` is correct, so let's make sure we spell it like that everywhere.

The `fresh-and-bright` template description also would need to be corrected.

**Comment by divyarajmasani** [comment]
@swissspidy this PR is good to go with your review. :)

**Review by swissspidy — APPROVED** [review]
Looks good to me!

**Pull request merged**